lib/CoreDisTools/coredistools.cpp

```diff
@@ -412,6 +412,37 @@ bool CorDisasm::decodeInstruction(BlockIterator &BIter, bool MayFail) const {
   return IsDecoded;
 }
 
+static bool tryDecodePCRelLabel(MCInst const& Inst, int64_t& Disp) {
+  constexpr unsigned int AArch64_LDRDl = 2359;
+  constexpr unsigned int AArch64_LDRSl = 2407;
+
+  bool HasPCRelLabel      = false;
+  unsigned int OperandNum = -1;
+
+  switch (Inst.getOpcode()) {
+    case AArch64_LDRDl:
+    case AArch64_LDRSl:
+      HasPCRelLabel = true;
+	  OperandNum    = 1;
+	  break;
+  }
+
+  if (HasPCRelLabel) {
```

Comment:
Can't it be replaced with `Inst.getOpcode() == AArch64_LDRDl || Inst.getOpcode() == AArch64_LDRSl`?